testsuite/test_wbemconnection_mock.py

```diff
@@ -1559,11 +1667,12 @@ def test_getclass(self, conn, tst_class):
 
     @pytest.mark.parametrize(
         "ns, cln, tst_ns, exp_er", [
-            [None, 'CIM_Foo', None, None],
-            [None, 'cim_foo', None, None],
-            [None, 'CIM_Foo', 'root/blah', CIM_ERR_INVALID_NAMESPACE],
-            ['root/blah', 'CIM_Foo', 'root/blah', None],
-            [None, 'CIM_Foox', None, CIM_ERR_NOT_FOUND],
+            [DEFAULT_NAMESPACE, 'CIM_Foo', None, None],
+            [DEFAULT_NAMESPACE, 'cim_foo', None, None],
+            [DEFAULT_NAMESPACE, 'CIM_Foo', 'BadNamespace',
+             CIM_ERR_INVALID_NAMESPACE],
```

Comment:
Doesn't this actually change the test? Most of the tests were None and one was a namespace?  This has namespace DEFAULT_NAMESPACE for all of the tests. I have not looked elsewhere to see if the same question existsyet.

Reply:
The `ns` parameter is used as the namespace in which the CIM objects are set up. The `tst_ns` parameter is the namespace against which the operation to be tested (GetClass in this case) is run. It does not make sense to set up CIM objects during test preparation n a namespace `None`, if that is not the purpose of the test.